experiments/omip_prototype/quarter_degree_omip.jl

```diff
@@ -0,0 +1,123 @@
+using ClimaOcean
+using ClimaSeaIce
+using Oceananigans
+using Oceananigans.Grids
+using Oceananigans.Units
+using Oceananigans.OrthogonalSphericalShellGrids
+using ClimaOcean.OceanSimulations
+using ClimaOcean.ECCO
+using ClimaOcean.DataWrangling
+using ClimaSeaIce.SeaIceThermodynamics: IceWaterThermalEquilibrium
+using Printf
+
+using CUDA
+CUDA.device!(1)
+arch = GPU()
+
+r_faces = ClimaOcean.exponential_z_faces(; Nz=100, h=30, depth=6200)
+z_faces = MutableVerticalDiscretization(r_faces)
+
+Nx = 1440 # longitudinal direction 
+Ny = 700  # meridional direction 
+Nz = length(r_faces) - 1
+
+grid = TripolarGrid(arch;
+                    size = (Nx, Ny, Nz),
+                    z = z_faces,
+                    halo = (7, 7, 7))
+
+bottom_height = regrid_bathymetry(grid; minimum_depth=15, major_basins=1)
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height); active_cells_map=true)
+
+#####
+##### A Propgnostic Ocean model
+#####
+
+# A very diffusive ocean
+momentum_advection = WENOVectorInvariant()
+tracer_advection   = WENO(order=7)
+
+free_surface = SplitExplicitFreeSurface(grid; substeps=70)
+closure = ClimaOcean.OceanSimulations.default_ocean_closure()
+
+ocean = ocean_simulation(grid;
+                         momentum_advection,
+                         tracer_advection,
+                         free_surface,
+                         closure)
+
+dataset = ECCO4Monthly()
+
+set!(ocean.model, T=Metadatum(:temperature; dataset),
+                  S=Metadatum(:salinity;    dataset))
+
+#####
+##### A Prognostic Sea-ice model
+#####
+
+# Remember to pass the SSS as a bottom bc to the sea ice!
+SSS = view(ocean.model.tracers.S.data, :, :, grid.Nz)
+bottom_heat_boundary_condition = IceWaterThermalEquilibrium(SSS)
+
+# Default dynamics
+dynamics = ClimaOcean.SeaIceSimulations.default_sea_ice_dynamics(grid; ocean)
+
+sea_ice = sea_ice_simulation(grid; bottom_heat_boundary_condition, dynamics, advection=WENO(order=7))
+
+set!(sea_ice.model, h=Metadatum(:sea_ice_thickness;     dataset),
+                    ℵ=Metadatum(:sea_ice_concentration; dataset))
+
+#####
+##### A Prescribed Atmosphere model
+#####
+
+atmosphere = JRA55PrescribedAtmosphere(arch; backend=JRA55NetCDFBackend(40))
+radiation  = Radiation()
+
+#####
+##### Arctic coupled model
+#####
+
+omip = OceanSeaIceModel(ocean, sea_ice; atmosphere, radiation)
+omip = Simulation(arctic, Δt=1minutes, stop_time=30days)
```

Comment:
I guess it not `arctic` but `omip`? there is a re-definition in this line then? `omip` above is a coupled model and here it becomes a simulation of coupled model?

Reply:
right, I will push the new script